### PR TITLE
Simplify repository and update service package

### PR DESCRIPTION
**Pull request by vbfox**
This change reuse heavily the new documentation for the [wdio-service](https://www.npmjs.com/package/@waldoapp/wdio-service) package and will require small changes to the in-app guide (WebApp).

With this new version the setup is aligned with the default scafolding of the WebDriverIO project and easier to reuse.

It also add the few files that were remaining in the private repository only like the GitHub workflow for linting as there is now no need for the split repository structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,152 +86,73 @@ It describes the instructions below, as well as shows you how to edit a script.
...
 ```
 
-## Work on your first test
-
-### Upload your app
+## Exploring Waldo Scripting
 
-Since Waldo Core API operates on devices running within the Waldo infrastructure, the first step to
-be able to test your app is to upload it to Waldo.
+Once you have run the sample scripts, you can explore explore the
```
Comment (VincentJaouen) [inline README.md:110]:
```suggestion
Once you have run the sample scripts, you can explore the
```

File: README.md [hunk 2]
```diff
@@ -1,51 +1,69 @@
...
-Refer to the [nodejs.org installation instructions](https://nodejs.org/en/learn/getting-started/how-to-install-nodejs) for more information.
+Refer to the [nodejs.org installation instructions](https://nodejs.org/en/learn/getting-started/how-to-install-nodejs)
+for more information.
 
 ## Getting started
 
-The best way to discover scripting with Waldo is to follow our guide in the product:
+> [!IMPORTANT]
+> The samples uses the [@wdio/wdio-service](https://www.npmjs.com/package/@waldoapp/wdio-service) package to
+> communicate with Waldo Scripting. The package README contains more information about the service, how to configure
```
Comment (VincentJaouen) [inline README.md:58]:
```suggestion
> communicate with Waldo Scripting. The package's README contains more information about the service, how to configure
```

File: README.md [hunk 3]
```diff
@@ -68,152 +86,73 @@ It describes the instructions below, as well as shows you how to edit a script.
...
 
-- `VERSION_ID` is the `id` of a specific build file of an app (hence the “app version”) you want to target with your test.
-  See the [upload section](#upload-your-app) of the documentation.
+### Integrating in your own project
 
-- `SHOW_SESSION` enables [live execution](#live-execution).
+If you already have [a WebDriverIO project][existing-wdio] or want to [start from scratch][from-scratch] you can
+follow the corresponding sections of the service readme.
 
-- `SESSION_ID` enables [interactive execution](#interactive-execution).
```
Comment (VincentJaouen) [inline README.md:215]: Is `SHOW_SESSION` mentioned anywhere now?

**Review by VincentJaouen — COMMENTED** [review]
Changes look good but I think they should still be done in https://github.com/waldoapp/Wikipedia-scripted-testing

File: README.md (hunk 3 above)
Comment (vbfox) [reply README.md:215]: It's mentioned in https://www.npmjs.com/package/@waldoapp/wdio-service#live-execution page and documented in https://www.npmjs.com/package/@waldoapp/wdio-service#showsession

And also present in the code sample of the "Running the sample scripts" section ?

**Review by VincentJaouen — APPROVED** [review]
LGTM

**Pull request merged**